modules/vmss/main.tf

```diff
@@ -17,6 +17,13 @@ resource "azurerm_linux_virtual_machine_scale_set" "this" {
   zones                           = var.zones
   zone_balance                    = var.zone_balance
   provision_vm_agent              = false
+  dynamic "admin_ssh_key" {
+    for_each = var.ssh_key == null ? [] : [1]
```

Comment:
why do we rely here on a single key? and not on a list of keys like in the `vmseries` module?

Reply:
Apologies - this was was left from a few months ago when my knowledge on the subiect was sparse - I updated it now.